### PR TITLE
Added readme to describe the goals for the test plan

### PR DESCRIPTION
**Pull request by mnm678**
Added a description of what will be included in the test plan (MUSTs and SHALLs from the standard) and what will not (compliance with the Uptane Standard, implementation details).

### CODE REVIEW

**Comment by zabbal** [comment]
Good to see this rolling, +1 for merging this (possibly with suggested changes on top).

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Uptane Test Plan
+This repository hosts the Uptane Test Plan. This test plan is intended to help Uptane adopters demonstrate that their implementation follows the requirements of the Uptane Standard.
+
+A test plan will be prepared for each published version of the Uptane Standard listing the requirements for compliance with that version of the Standard.
+
+## Goals
+The goal of this test plan is to provide a framework for verifying that an implementation meets all of the MUSTs and SHALLs stated in the Uptane Standard.
+
+## Non-Goals
+Results of the tests contained in this plan do not confer any type of official certification, or assure compliance with the Uptane Standard.
```
Comment (jhdalek55) [inline README.md]:
```suggestion
Results of the tests contained in this strategy do not confer any type of official certification, or assure compliance with the Uptane Standard.
```

**Comment by jhdalek55** [comment]
@JustinCappos @trishankkarthik or @patrickvacek--can one or two of you review and comment so this can be merged? It's been open almost two and a half months.

**Comment by jhdalek55** [comment]
Actually, @zabbal did OK this, so we might be able to merge now.

**Review by JustinCappos — APPROVED** [review]
LGTM

**Pull request merged**